src/wwwroot/controllers/BillingCtrl.js

```diff
@@ -121,6 +128,16 @@
         return;
       }
     }
+
+    function fillList() {
+      var lang = $translate.use();
+
+      countriesPromise.then(function(ret) {
+        vm.countryList = ret.data.map(function(val){
+          return { code: val.code, name: val[lang] };
+        })
+      });
+    }
```

Comment:
Maybe we could move all this common country data to a single place, it could avoid registration and deregistration complications